testutils/testutils.go

```diff
@@ -122,13 +141,17 @@ func MockQueryStateResponse() jsonrpc.ResponseQueryState {
 	partialState := pack.NewStruct(
 		"pubKey", pack.String("Akwn5WEMcB2Ff_E0ZOoVks9uZRvG_eFD99AysymOc5fm"),
 	)
-
+	systemStateP, err := pack.Encode(MockSystemState())
+	if err != nil {
+		panic("Shouldn't fail")
```

Comment:
```suggestion
		panic("encoding state: %v", err)
```

Probably still worth printing the error just in case.